src/shared/components/simpleTable/SimpleTable.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from 'react';
+import './styles.scss';
+import {observable, computed, action} from "mobx";
+import {observer} from "mobx-react";
+
+export type Column = {
+    name:string;
+    sortable?:boolean;
+};
+
+export type Row = { [columnName:string]:JSX.Element };
+
+
+type SimpleTableProps = {
+    columns:Column[];
+    rows:Row[];
+    onSort?:(name:string, sortAscending:boolean)=>void;
+};
+
+class SimpleTableStore {
+    private onSort?:(name:string, sortAscending:boolean)=>void;
+
+    @observable public sortColumn:string;
+    @observable public sortAscending:boolean;
+    @observable.ref public rows:Row[];
+    @observable public columns:Column[];
+    @computed get thElements():JSX.Element[] {
```

Comment:
just fyi, i think we're going to need to support custom column headers.  andy has a use case for this and asked me.